verticapy/vdataframe.py

```diff
@@ -11887,16 +11923,31 @@ def to_json(
             for row in result:
                 tmp_row = []
                 for i, item in enumerate(row):
-                    if isinstance(item, (float, int, decimal.Decimal)):
+                    if isinstance(item, (float, int, decimal.Decimal)) or (
+                        isinstance(item, (str,)) and is_complex_vmap[i]
+                    ):
                         tmp_row += ["{}: {}".format(quote_ident(columns[i]), item)]
                     elif item != None:
                         tmp_row += ['{}: "{}"'.format(quote_ident(columns[i]), item)]
-                file.write("{" + ", ".join(tmp_row) + "},\n")
+                json_file += "{" + ", ".join(tmp_row) + "},\n"
             current_nb_rows_written += limit
             file_id += 1
-            file.write("]")
-            file.close()
-        return self
+            json_file += "]"
+            if n_files == 1 and path:
+                file = open(path, "w+")
+                file.write(json_file)
+                file.close()
+            elif path:
+                file = open("{0}/{1}.json".format(path, file_id), "w+")
```

Comment:
The same as above. 
There is already "" outside of {0}. And path is also a string. It may lead to ""/folder/subfolder"/".

Reply:
Not true, path is not modified and '"' are Python quotes and do not belong to the string.